RELEASING.md

```diff
@@ -69,10 +69,26 @@ For a discussion on whether to add a prefixing `v` to release tags, see [#35](ht
 
 Repositories create consistent release labels, such as `v1.0.0`, `v1.1.0` and `v2.0.0`, as well as `patch` and `backport`. Use release labels to target an issue or a PR for a given release. See [MAINTAINERS](MAINTAINERS.md#triage-open-issues) for more information on triaging issues.
 
-## Releasing
+## Release Scheduling
 
-The OpenSearch release process is centralized.
+The OpenSearch release process is centralized and has several steps along the way.  As part of this process, the Engineering Efficiency team will cut a release ticket and assign an overall release manager for the release.  They will also cut individual tickets per repo for the release with what's expected to go in that release.   
```

Comment:
Calling out this team is good, how should reader of this document get in touch with them?  We could make the team public and add a link to them.

Reply:
Oof.  So as far as I can tell, even if the team is public, if you're not a member of the organization you can't seem them (this has bitten me before).    Maybe if we suggest folks @ the team in the ticket to get the team's attention?

Reply:
> Maybe if we suggest folks @ the team in the ticket to get the team's attention?

Works well for me